monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -109,7 +115,6 @@ class ReportPageComponent extends AuthComponent {
       },
       'tunnel': {
         [this.issueContentTypes.OVERVIEW]: tunnelIssueOverview,
-        [this.issueContentTypes.REPORT]: tunnelIssueReport,
         [this.issueContentTypes.TYPE]: this.issueTypes.WARNING
```

Comment:
Remove this whole thing, if we turn overview and report into standalone components, they can determine by machines and agents if they should render or not